README.md

```diff
@@ -14,21 +14,21 @@ Apache-2.0 License. See [LICENSE](LICENSE) file for details.
 ## Installation
 
 ```bash
-pip install ttyg-evaluation
+pip install qa-eval
 ```
 
 ## Maintainers
 
 Developed and maintained by [Graphwise](https://graphwise.ai/).
-For issues or feature requests, please open [a GitHub issue](https://github.com/Ontotext-AD/ttyg-evaluation/issues).
+For issues or feature requests, please open [a GitHub issue](https://github.com/Ontotext-AD/qa-eval/issues).
 
 ## Usage
 
-To use this module you must provide a gold standard corpus that defines questions and expected tool calls for each question.
+To use this module you must provide a reference corpus that defines questions and (optionally) the expected tool calls for each question.
 
-### Gold Standard Format
+### Reference Standard Format
 
-A gold standard corpus is a list of templates. Each template contains:
+A reference corpus is a list of templates, each of which contains:
```

Comment:
Similar comment here, I was under the impression we will ditch `corpus` and replace it with `dataset`